README.md

```diff
@@ -2,53 +2,25 @@
 
 # Hello Microverse
 
-> This project of hello microverse project about github flow.
-
-![screenshot](./app_screenshot.png)
+> This is a basic project called Hello Microverse about basic github flow.
 
 
```

Comment:
- [ ] **OPTIONAL:** Please do not merge the feature branch with any changes to the main branch until you get approval in next projects, because it will cause problems when you are working collaboratively